src/main/java/com/iexec/common/docker/client/DockerClientInstance.java

```diff
@@ -563,7 +558,7 @@ public HostConfig buildHostConfigFromRunRequest(DockerRunRequest dockerRunReques
                     dockerRunRequest.getBinds().toArray(new String[0])));
         }
         if (dockerRunRequest.getDevices() != null) {
```

Comment:
Maybe add a variable to avoid calling twice `dockerRunRequest.getDevices()`?

Reply:
See https://github.com/iExecBlockchainComputing/iexec-common/pull/296/commits/5b50661cb8f1122e8edac64d1d06805a1c6c5941